includes/CLI/Plugin_Check_Command.php

```diff
@@ -248,6 +250,15 @@ static function ( $dirs ) use ( $excluded_files ) {
 		$error_severity   = ! empty( $options['error-severity'] ) ? $options['error-severity'] : $options['severity'];
 		$warning_severity = ! empty( $options['warning-severity'] ) ? $options['warning-severity'] : $options['severity'];
 
+		$format_is_wporg = ( isset( $options['format'] ) && 'wporg' === $options['format'] );
```

Comment:
```suggestion
		$is_wporg_format = isset( $options['format'] ) && 'wporg' === $options['format'];
```

Nit. name should be `is_wporg_format`

Reply:
ok!